browser-extension/mv2/src/client/js/sessionRecorder.js

```diff
@@ -113,18 +113,25 @@ RQ.SessionRecorder.addMessageListeners = () => {
       RQ.SessionRecorder.sendResponseToRuntime(event.data.action, event.data.payload);
     } else if (event.data.action === "sessionRecordingStarted") {
       RQ.SessionRecorder.isRecording = true;
+
+      if (!RQ.SessionRecorder.showWidget) return;
+
```

Comment:
It should be after `NOTIFY_SESSION_RECORDING_STARTED` event. It would more readable if the show widget conditions are wrapped in `RQ.SessionRecorder.showWidget`